frontend/src/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -100,7 +102,8 @@ const EmployeeModal = ({ existingEmployee }: EmployeeModalProps) => {
     )
       .then(() => {
         refresh();
-        setToast(isCreate);
+        isCreate ? showToast('Showing Toast') : null;
```

Comment:
use an if statement here for readability, and change the toast text to what it should be

Reply:
also is this necessary? since we're already showing toasts after an employee is created/edited, we dont need another after the photo is uploaded